src/winetricks

```diff
@@ -721,11 +721,11 @@ w_expand_env()
 # Get the latest tagged release from github.com API
 w_get_github_latest_release()
 {
-    # $1 - github organization
-    # $2 - github repository
     # FIXME: can we get releases that aren't on master branch?
+    org="$1"
+    repo="$2"
 
-    WINETRICKS_SUPER_QUIET=1 w_download_to "${W_TMP_EARLY}" "https://api.github.com/repos/$1/$2/releases/latest" "" "release.json" >/dev/null 2>&1
+    WINETRICKS_SUPER_QUIET=1 w_download_to "${W_TMP_EARLY}" "https://api.github.com/repos/${org}/${repo}/releases/latest" "" "release.json" >/dev/null 2>&1
```

Comment:
`https://api.github.com/repos/${org}/${repo}/releases/latest`  -> https://api.github.com/repos/wine-mirror/wine/releases/latest which results in error?

Reply:
nwm https://api.github.com/repos/winetricks/winetricks/releases/latest works -> doesn't work on mirrors